data/json/recipes/recipe_armor.json

```diff
@@ -9074,5 +9074,80 @@
         [["duct_tape", 35], ["superglue", 5]],
         [["jerrycan", 1]]
     ]
-  }
+  },{
+  "type" : "recipe",
+  "result": "wetsuit",
+  "category": "CC_ARMOR",
+  "subcategory": "CSC_ARMOR_SUIT",
+  "skill_used": "tailor",
+  "skills_required" : [ "tailor", 4 ],
```

Comment:
You declared tailor skill twice here. "skills_required" refers to secondary skills.